### ISSUE TITLE
v4+ includes dependencies

### ISSUE THREAD

**Issue by viktorix** [msiemens/tinydb#315]
When I was testing prior db.insert issue, I noticed dependencies were added to pipfile.lock. It started with v4.0.0. v3.15.2 doesn't have them.

```
tinydb==4.1.0
  - pytest-mypy [required: >=0.4.1,<0.5.0, installed: 0.4.2]
    - mypy [required: >=0.700, installed: 0.770]
      - mypy-extensions [required: >=0.4.3,<0.5.0, installed: 0.4.3]
      - typed-ast [required: >=1.4.0,<1.5.0, installed: 1.4.1]
      - typing-extensions [required: >=3.7.4, installed: 3.7.4.2]
    - pytest [required: >=2.8, installed: 5.4.1]
      - atomicwrites [required: >=1.0, installed: 1.4.0]
      - attrs [required: >=17.4.0, installed: 19.3.0]
      - colorama [required: Any, installed: 0.4.3]
      - more-itertools [required: >=4.0.0, installed: 8.2.0]
      - packaging [required: Any, installed: 20.3]
        - pyparsing [required: >=2.0.2, installed: 2.4.7]
        - six [required: Any, installed: 1.14.0]
      - pluggy [required: >=0.12,<1.0, installed: 0.13.1]
      - py [required: >=1.5.0, installed: 1.8.1]
      - wcwidth [required: Any, installed: 0.1.9]
```

**Comment by msiemens** [comment]
Hey! Seems like somehow optional dependencies are always installed with poetry when installing from PyPI. I've now moved these dependenies to `dev-dependencies` and published v4.1.1 which should not install these packages. Can you please confirm this?

**Comment by viktorix** [comment]
Works, thanks!
